compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -336,7 +337,11 @@ class ROCMTargetBackend final : public TargetBackend {
     mpm.addPass(llvm::VerifierPass());
     mpm.addPass(pb.buildPerModuleDefaultPipeline(ol));
     mpm.addPass(llvm::VerifierPass());
-
+    llvm::raw_string_ostream os(outPassesString);
+    mpm.printPipeline(os, [&pic](StringRef ClassName) {
+      auto PassName = pic.getPassNameForClassName(ClassName);
+      return PassName.empty() ? ClassName : PassName;
+    });
```

Comment:
lowercase for local variable names in IREE? I guess this code might have been copied from LLVM that capitalizes local variables